src/main/java/com/slimebot/main/Main.java

```diff
@@ -101,7 +101,8 @@ public static void main(String[] args) throws IOException {
 				.addEventListeners(new StartupListener())
 				.addEventListeners(new TimeoutListener())
 				.addEventListeners(new JoinListener())
-				.addEventListeners(new StaffMessage());
+				.addEventListeners(new StaffMessage())
+				.addEventListeners(new BugCommand());
```

Comment:
Don't add the command as listener but use `registerCommand(BugCommand.class)` instead

Reply:
Done